scripts/tests/relay-graceful-shutdown.sh

```diff
@@ -0,0 +1,25 @@
+#!/usr/bin/env bash
+
+set -euo pipefail
+
+source "./scripts/tests/lib.sh"
+
+# Arrange: Setup a relayed connection
+install_iptables_drop_rules
+client_curl_resource "172.20.0.100/get"
+
+# Act: Send SIGTERM
+sudo kill --signal SIGTERM "$(pgrep firezone-relay)"
+
+# Assert: Process is still there and dataplane still works
+pgrep firezone-relay
```

Comment:
This might be a bit redundant? What we actually care about is that the connection still works

Reply:
we should check if the docker container is still running right?

Reply:
I mean, for both this and the previous comment, if it works it's fine I'm just a bit confused

Reply:
> This might be a bit redundant? What we actually care about is that the connection still works

Yeah I think that is right, I'll remove it.